<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NSceneDialog</class>
 <widget class="QDialog" name="NSceneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QComboBox" name="transition">
     <property name="currentIndex">
      <number>6</number>
     </property>
     <item>
      <property name="text">
       <string>Navy.Transition.SlideRight</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Navy.Transition.SlideLeft</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Navy.Transition.SlideUp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Navy.Transition.SlideDown</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Navy.Transition.PopUp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Navy.Transition.TurnOver</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Navy.Transition.Fade</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="id"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>class</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>transition</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="className"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>id</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>b.g. color</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="backgroundColor"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="pageLabel">
     <property name="text">
      <string>page</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="layoutLabel">
     <property name="text">
      <string>layout</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="classFileLabel">
     <property name="text">
      <string>class file</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="NTextListSelector" name="classFile">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="NTextListSelector" name="layout">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="NTextListSelector" name="page">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NTextListSelector</class>
   <extends>QPushButton</extends>
   <header>extend/n_text_list_selector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
